module/core/src/Test/Behat/Context/ApiAuthContext.php

```diff
@@ -34,6 +34,7 @@ public function __construct(
     public function iAmAuthenticatedAsUser(User $user): void
     {
         $token = $this->JWTTokenManager->create($user);
+        /** @phpstan-ignore-next-line */
```

Comment:
Why do u ignore next line?

Reply:
There is no such function like 'setHttpHeader' on Request object, there is used magic method __call and phpstan is too stupid to figure this out ;)